core/.gas-snapshot

```diff
@@ -22,5 +22,5 @@ LiquidatorGasTest:test_noCallbackTwoAssets() (gas: 52073)
 LiquidatorGasTest:test_noCallbackTwoAssetsAndUniswapPosition() (gas: 94964)
 LiquidatorGasTest:test_warn() (gas: 27875)
 LiquidatorGasTest:test_withCallbackAndSwap() (gas: 123133)
-VolatilityGasTest:test_consult() (gas: 13218)
+VolatilityGasTest:test_consult() (gas: 13142)
```

Comment:
Really not sure why this changed too

Reply:
Yeah, this is a bit weird considering that `test_consult` calls `oracle.update(pool)` and `oracle.consult(pool)` of which neither should be impacted by this change. The difference in gas seems negligible though.